src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -94,6 +100,10 @@ public String getTriggeredStatus() {
     public Boolean getAddTestResults() {
         return addTestResults == null ? Boolean.valueOf(false) : addTestResults;
     }
+    
+    public Boolean getUpdateQueuePosition() {
+        return updateQueuePosition == null ? false : updateQueuePosition;
```

Comment:
to be in line with the rest of the file you would need to replace `false` with `Boolean.valueOf(false)`